concent_api/core/exceptions.py

```diff
@@ -83,3 +83,7 @@ class BanksterTimestampError(Exception):
 
 class BanksterTransactionMismatchError(Exception):
     pass
+
+
+class MaxRetriesExceededError(ConcentBaseException):
```

Comment:
It inherits from `ConcentBaseException` which is handled as `HTTP400` is really a bad request when `MaxRetriesExceededError`? I wouldn't say so.

Reply:
OK, changed. I did it to provide return message, but there is no need, Exception type is pretty obvious.